src/models/SqlOutputContentProvider.ts

```diff
@@ -475,7 +471,7 @@ export class SqlOutputContentProvider implements vscode.TextDocumentContentProvi
             }
         }
 
-        vscode.workspace.openTextDocument(uri).then((doc: vscode.TextDocument) => {
+        vscode.workspace.openTextDocument({ language: linkType }) .then((doc: vscode.TextDocument) => {
```

Comment:
It looks like having a bad language type won't cause a crash, but will instead default to 'Plain Text'. This isn't terrible or anything, but since we are already checking above if we have 'xml' or 'json' it might be a good idea to not launch this unless we have 'xml' or 'json'.